### PR TITLE
#444 : Adding section

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: training/030_dtl/030_Novice.rst [hunk 1]
```diff
@@ -294,3 +294,57 @@ Type eksempler:
...
+
+#. *What is ``["apply"]`` categorized as?*
+
+#. *Name the two ways in which "Merge as a function" can be used?*
+
+#. *What does ``["hops"]`` allow you to do?*
+
+#. *What are ``"_"`` properties categorized as?*
+
+#. *Name four type examples that exist in Sesam?*
```
Comment (ga-hegsvold) [inline training/030_dtl/030_Novice.rst:328]: "type" as in "data type"?

File: training/030_dtl/030_Novice.rst [hunk 2]
```diff
@@ -294,3 +294,57 @@ Type eksempler:
...
+
+	In the upstream pipe use:
+
+		- ``["copy"]``
+
+		- ``["make-ni"]``
+
+	In the global pipe:
+
+		- Merge at least three namespaces
```
[the remark this answers is not in the document]
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:340]: what would you call it? not sure the dataset term is that relevant any longer??.. outputs?

File: training/030_dtl/030_Novice.rst (hunk 1 above)
Comment (ga-hegsvold) [reply training/030_dtl/030_Novice.rst:328]: Right, then maybe rephrase to "Name four data types supported by Sesam".

File: training/030_dtl/030_Novice.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (ga-hegsvold) [reply training/030_dtl/030_Novice.rst:340]: I'd say the term dataset is still relevant. Sesam still stores data in datasets, they are just not so visible anymore.

What I wanted to clarify with this sentence, thou, was the "merge ... three namespaces" part.
It might be a nit-pick but it sounds like the reader is instructed to merge the namespaces together.
I think its more precise to say that it's the data from the different namespaces that should be merged.
The sentence I wrote above was a suggested rephrasing.
What do you think?

File: training/030_dtl/030_Novice.rst (hunk 1 above)
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:328]: Will do. 👍

File: training/030_dtl/030_Novice.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:340]: ahh. See your point. I like that aspect. Will implement.

File: training/030_dtl/030_Novice.rst [hunk 3]
```diff
@@ -294,3 +294,57 @@ Type eksempler:
...
+
+		- Check for relations between the merged namespaces.
+
+			- In case of relations, define equalities.
```
[the remark this answers is not in the document]
Comment (JonasAls89) [reply training/030_dtl/030_Novice.rst:344]: well, that was why I stated it as I did. I aimed for making it more vague, so that we would avoid having to talk about key relations, albeit might not be optimal no... Same-ness to me is a bit fluffy yea, so will not do in this awesome book we are writing.. hmm.. what about: "overlap" or "associated" or simply "linked"??

[the remark this answers is not in the document]
Comment (ga-hegsvold) [reply training/030_dtl/030_Novice.rst:344]: After some discussions we have a couple of suggestions on how to address entities from different sources that are the same (i.e. a particular person represented in multiple different system):
"contextually equal"
"contextually the same"

Would like feedback on this.

**Review by ga-hegsvold — APPROVED** [review]
